### PR TITLE
Add ajax form

### PR DESCRIPTION
**Pull request by myrridin**
Results html for success and error were extracted into partials
Added public directory for js and images
Zepto (minimal jQuery) pulled in
On submit event added to form to render partials to the results div
Loading gif added

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -14,6 +14,7 @@ app.use('/js', express.static('js'));
 app.use('/normalize-css', express.static('bower_components/normalize-css'));
 app.use('/foundation/css', express.static('bower_components/foundation/css'));
 app.use('/foundation/js', express.static('bower_components/foundation/js'));
+app.use(express.static('public'));
```
Comment (jenkoian) [inline index.js]: Can you  move the css directory inside of public and then remove lines 12 & 13?

File: views/index.hbs [hunk 2]
```diff
@@ -11,6 +11,9 @@
     <link rel="stylesheet" href="/foundation/css/foundation.min.css">
 
     <link rel="stylesheet" href="/css/main.css">
+
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/zepto/1.1.6/zepto.min.js"></script>
+    <script src="/js/script.js"></script>
```
Comment (jenkoian) [inline views/index.hbs]: Could you move this to the bottom just above the GA code?

**Comment by jenkoian** [comment]
Thanks! Love the loading gif too :) Few minor things and will be good for merge :+1:

**Comment by jenkoian** [comment]
Oh another minor thing, could you add a tick on the README for ajax form :)

**Comment by myrridin** [comment]
All set. Thanks for the notes! Love the project.

**Comment by jenkoian** [comment]
Thanks!

**Pull request merged**